### PR TITLE
Badge add

### PR DESCRIPTION
**Pull request by berkatil**
I forgot to create a new branch for getting a badge point, sorry for this.
In this pull request, followings are the work done:

- Unit test is added for badge post
- Get badge point by name endpoint is added.
- Some comments are added.

### CODE REVIEW

**Review by eye-78 — CHANGES_REQUESTED** [review]
test file seems a bit short, you may expand it later
please edit views.py file in an appropriate way

File: practice-app/website/test/badge_post_test.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    def test_badges(self):
+        expected = Badge(name=self.data['name'], symbol=self.data['symbol'])
+        badge = prepare_badge(self.data)
+
+        self.assertEqual(expected.name, badge.name)
+        self.assertEqual(expected.symbol, badge.symbol)
+    
+    def test_badge_point(self):
+        expected = extract_point(self.bitcoin_price)
+        self.assertEqual(expected,'3') # 3 is the 4th digit after decimal point
```
Comment (MercerFrey) [inline practice-app/website/test/badge_post_test.py:20]: This test format looks good. I will create my unit test like this. Thanks.

**Review by MercerFrey — COMMENTED** [review]
Looks fine. I think it is ready to be merged.

**Comment by berkatil** [comment]
> test file seems a bit short, you may expand it later
> please edit views.py file in an appropriate way

thanks! I fixed view.py and I will look at tests later

File: practice-app/website/test/badge_post_test.py (hunk 1 above)
Comment (kataparuto) [reply practice-app/website/test/badge_post_test.py:20]: Yes, it might a nice starting point for you. This is a really clear test format.

**Pull request merged**